account_permanent_lock_move/wizard/permanent_lock_date_wizard.xml

```diff
@@ -0,0 +1,19 @@
+<odoo>
+    <record id="permanent_lock_date_wizard_form" model="ir.ui.view">
```

Comment:
not approve relevant, but would be nice to have id pattern like

> For a view: <model_name>_view_<view_type>, where view_type is kanban, form, tree, search, ...

Reply:
Done

Thanks for the review